examples/pigweed-app/esp32/main/main.cpp

```diff
@@ -24,33 +24,45 @@
 #include "pw_rpc/echo_service_nanopb.h"
 #include "pw_rpc/server.h"
 #include "pw_sys_io/sys_io.h"
-#include "pw_sys_io_esp32/init.h"
 
+#include "PigweedLoggerMutex.h"
 #include "RpcService.h"
+#include <support/logging/CHIPLogging.h>
 
 #include "lwip/err.h"
 #include "lwip/sockets.h"
 #include "lwip/sys.h"
 
-const char * TAG = "chip-pigweed-app";
+static const char * TAG = "chip-pigweed-app";
```

Comment:
nit: Prefer unnamed namespaces to `static` qualifiers.

Reply:
Removed static qualifier.